charts/bulk-scan-end-to-end-tests/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for bulk-scan-end-to-end-tests App
+name: bulk-scan-bulk-scan-end-to-end-tests
+home: https://github.com/hmcts/bulk-scan-end-to-end-tests
+version: 0.0.1
+maintainers:
+  - name: HMCTS bulk-scan team
```

Comment:
do we need a chart?

Reply:
don't think we need charts and related config